PrincessBrideTrivia/PrincessBrideTrivia/Program.cs

```diff
@@ -24,7 +24,7 @@ public static void Main(string[] args)
 
         public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
         {
-            return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+            return  ((( (float) numberCorrectAnswers / (float) numberOfQuestions) * 100).ToString()) + "%";
```

Comment:
Looks great! All tests pass and program runs as expected. Great idea for the EC portion. The only revision I can think of is cleaning the GetPercentCorrect method with this:
```suggestion
          return ((float)numberCorrectAnswers / numberOfQuestions * 100) + "%";
```
Typecasting the numerator as a float allows for the correct result. ToString() is unnecessary as the concatenation of "%" will automatically convert the the given float to a string.

Pull Request title includes "Assignment 1" in the title ✔
Pull Request targets Assignment 1 branch ✔
Issue 1: Application no longer crashes ✔
Issue 2: Unit test properly passes ✔
Extra Credit
Issue 3: Feature request implemented ✔
Issue 3: Feature request unit tested ✔

Reply:
Agreed that this could be cleaned up, but I do like how the output is formatted to the whole percent.